django_bitmask_field.py

```diff
@@ -1,14 +1,21 @@
 import codecs
 import functools
+import django
 
 from django import forms
 from django.core import checks, exceptions, validators
 from django.db import models
 from django.utils.encoding import force_bytes
-from django.utils.translation import ugettext_lazy as _
 from six import integer_types, text_type, PY3
 from six.moves import reduce
 
+django_version = django.VERSION
+if django_version[0] == 4:
```

Comment:
maybe it's better to check if Django version greater or equal 4?

Reply:
I thought about it but who nows what will happen with 5.x given that 4.1 is just out the door.
I'd cross that bridge when we come to it.